Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -223,4 +228,15 @@ void UIApplicationMainTest() {
     _ApplicationLaunch(ActivationTypeNone, nullptr);
 }
 
+UIKIT_EXPORT
+void UIApplicationActivationTest(IInspectable* activationArgs, wchar_t* delegateName){
```

Comment:
can we keep the interface similar to UIApplicationMain and use void* i.e. it takes a NSString* and because we cannot present that in C++/Cx file we use void*